src/compas_rhino/__init__.py

```diff
@@ -36,6 +36,8 @@
 PURGE_ON_DELETE = True
 
 INSTALLABLE_PACKAGES = ['compas', 'compas_rhino', 'compas_ghpython']
+SUPPORTED_VERSIONS = ['5.0', '6.0', '7.0', '8.0']
+DEFAULT_VERSION = '6.0'
```

Comment:
yes, would be more than happy to, but thought it might be considered a breaking change...

Reply:
good point... I did not consider it breaking change because it's not a runtime behavior... of course, it's debatable, but at least, if you have `compas` already installed, and you conda update it, nothing would break due to this...